.github/workflows/test.yml

```diff
@@ -21,7 +21,6 @@ jobs:
     - name: Install dependencies
       run: |
         pip install pytest
-        pip install mujoco==2.3.2
         git clone https://github.com/benelot/pybullet-gym.git
```

Comment:
I guess we can remove pybullet since the pybullet reacher was removed previously?